GPU/Common/PresentationCommon.cpp

```diff
@@ -60,54 +83,54 @@ void CenterDisplayOutputRect(float *x, float *y, float *w, float *h, float origW
 			float smallDisplayW = origW * customZoom;
 			float smallDisplayH = origH * customZoom;
 			if (!rotated) {
-				*x = floorf(((frameW - smallDisplayW) / 2.0f) + offsetX);
-				*y = floorf(((frameH - smallDisplayH) / 2.0f) + offsetY);
-				*w = floorf(smallDisplayW);
-				*h = floorf(smallDisplayH);
+				rc->x = floorf(((frame.w - smallDisplayW) / 2.0f) + offsetX);
+				rc->y = floorf(((frame.h - smallDisplayH) / 2.0f) + offsetY);
+				rc->w = floorf(smallDisplayW);
+				rc->h = floorf(smallDisplayH);
 				return;
 			} else {
-				*x = floorf(((frameW - smallDisplayH) / 2.0f) + offsetX);
-				*y = floorf(((frameH - smallDisplayW) / 2.0f) + offsetY);
-				*w = floorf(smallDisplayH);
-				*h = floorf(smallDisplayW);
+				rc->x = floorf(((frame.w - smallDisplayH) / 2.0f) + offsetX);
+				rc->y = floorf(((frame.h - smallDisplayW) / 2.0f) + offsetY);
+				rc->w = floorf(smallDisplayH);
+				rc->h = floorf(smallDisplayW);
 				return;
 			}
 		} else if (g_Config.iSmallDisplayZoomType == (int)SmallDisplayZoom::AUTO) {
 			// Stretch to 1080 for 272*4.  But don't distort if not widescreen (i.e. ultrawide of halfwide.)
-			float pixelCrop = frameH / 270.0f;
+			float pixelCrop = frame.h / 270.0f;
 			float resCommonWidescreen = pixelCrop - floor(pixelCrop);
-			if (!rotated && resCommonWidescreen == 0.0f && frameW >= pixelCrop * 480.0f) {
-				*x = floorf((frameW - pixelCrop * 480.0f) * 0.5f);
-				*y = floorf(-pixelCrop);
-				*w = floorf(pixelCrop * 480.0f);
-				*h = floorf(pixelCrop * 272.0f);
+			if (!rotated && resCommonWidescreen == 0.0f && frame.w >= pixelCrop * 480.0f) {
+				rc->x = floorf((frame.h - pixelCrop * 480.0f) * 0.5f);
```

Comment:
@hrydgard rc->x should use frame.w

Reply:
Doh! Good catch.

Reply:
Sorry for missing, thanks for the catch.

-[Unknown]